README.md

```diff
@@ -2,9 +2,22 @@
 
 Elm support for Sublime's LSP plugin.
 
+
+### Requirements 
+
+Make sure you have [NodeJS and NPM](https://nodejs.org/en/) installed.
+
+You will need to install `elm` and `elm-test` to get all diagnostics and `elm-format` for formatting. 
+
+```
+npm install -g elm elm-test elm-format
+```
+
 * Install [LSP](https://packagecontrol.io/packages/LSP), [Elm Syntax Highlighting](https://packagecontrol.io/packages/Elm%20Syntax%20Highlighting) and `LSP-elm` from Package Control.
```

Comment:
Maybe this could have its own `### Installation` section?

Reply:
I just found out that I can automate that.
I can set the `elmPath`, `elmTestPath` and elmFormatPath in the initializationOptions
https://github.com/elm-tooling/elm-language-server#server-settings

One more thing that someone doesn't have to do :)

I just need some way to specify something like this (intelephense does something like this):
"elmPath": "${cache}/LSP-elm/node_modules/bin/elm"

Reply:
I was planning to put that variable expanding code in lsp_utils. Even made the code.

Reply:
But I won't do that now :)

Reply:
Although I wasn't happy with the fact that we have to use a global sublime window to get variables so wanted someone to do some refactoring in LSP first. :)

Reply:
But in any case, I think you can postpone that to another PR.

Reply:
Probably, I am leaving it this way, 
so that the elm version that this plugin uses, 
doesn't end up being different version from the one that someone is using to compile the code.

Reply:
Thanks for the review :)